src/components/elements/__tests__/Alert.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { StyleSheetTestUtils } from 'aphrodite'
+import { render } from 'enzyme'
+import { Alert, Button } from '..'
```

Comment:
for tests, we should avoid importing and evaluating extra code, so import statements in all tests should be made this way: ```import Alert from '../Alert'```